registry/zookeeper/registry.go

```diff
@@ -381,7 +381,7 @@ func (r *zkRegistry) registerTempZookeeperNode(root string, node string) error {
 	}
 	zkPath, err = r.client.RegisterTemp(root, node)
 	if err != nil {
-		logger.Errorf("RegisterTempNode(root{%s}, node{%s}) = error{%v}", root, node, perrors.WithStack(err))
+		logger.Warnf("RegisterTempNode(root{%s}, node{%s}) = error{%v}", root, node, perrors.WithStack(err))
 		return perrors.WithMessagef(err, "RegisterTempNode(root{%s}, node{%s})", root, node)
```

Comment:
I think not all error's log level need set to warn. You can specify the 'node already exists' error type and set its log level to warn.

Reply:
i will fix it